fileinfo.go

```diff
@@ -13,8 +13,32 @@ import (
 //sys setFileInformationByHandle(h syscall.Handle, class uint32, buffer *byte, size uint32) (err error) = SetFileInformationByHandle
 
 const (
-	fileBasicInfo = 0
-	fileIDInfo    = 0x12
+	// FILE_INFO_BY_HANDLE_CLASS
```

Comment:
Rather than redefining these all here, this might be a good opportunity to just move to using the definitions from here: https://pkg.go.dev/golang.org/x/sys/windows#FileStandardInfo

Reply:
Oh! I had no idea those were there. Yeah, _much_ better.

Reply:
This turned into an adventure, because those constants needed a newer version of `golang.org/x/sys/windows` than we had. And that also gave us implementations of the syscalls themselves, rather than having to generate them here.

And then I got excited.